package.json

```diff
@@ -50,7 +49,7 @@
   "husky": {
     "hooks": {
       "pre-push": "npm test",
-      "pre-commit": "pretty-quick --staged"
```

Comment:
This was trying to use the default prettier parser on files containing TypeScript specific syntax and was failing as a result.